powersimdata/input/tests/test_transform_profile.py

```diff
@@ -86,8 +85,13 @@ def get_change_table_for_new_plant_addition(base_grid, resource):
 
 
 def _check_plants_are_scaled(ct, base_grid, base_profile_resource, resource):
+    plant_id_type = base_grid.plant.groupby("type").get_group(resource).index
+
     profile_info = base_profile_resource[0]
-    base_profile = base_profile_resource[1]
+    base_profile = (
+        base_profile_resource[1][plant_id_type]
+        * base_grid.plant.loc[plant_id_type, "Pmax"]
+    )
```

Comment:
I see what's going on here with `base_profile_resource`, but it's a little confusing. What do you think about changing the params for `_check_plants_are_scaled` to (ct, base_grid, **profile_info, base_profile**, resource), and then calling this function with e.g. `_check_plants_are_scaled(ct, base_grid, *base_solar, "solar")`? Then `base_solar` unpacks automatically and the call signature for this check is a bit clearer.

I think we can also simplify `_ = _check_plants_are_scaled(...)` to just `_check_plants_are_scaled(...)`.

Reply:
The other option would be to scale the profiles (not demand) in their respective fixture so you always get the normalized profile scaled by the Pmax of the plants

Reply:
My comment was less about where you do the profile scaling, more about not wanting the `base_profile_resource` parameter to be a tuple that is immediately unpacked, when it would be clearer to have two parameters instead.

Reply:
Done